bin/runsvdir-dyno

```diff
@@ -14,6 +14,31 @@ SERVICE_DIR=".service.$PS_TYPE"
 rm -rf "$SERVICE_DIR"
 mkdir "$SERVICE_DIR"
 
-bin/runsvdir-populate-service-dir "$SERVICE_DIR" "$PS_PROCFILE"
+cat "$PS_PROCFILE" | \
```

Comment:
Shellcheck complains about a useless use of `cat` but I think this is clearer than piping the file into `read` using `<`.